v3.3/glfw/glfw.go

```diff
@@ -35,9 +35,13 @@ const (
 func Init() error {
 	C.glfwInit()
 	// invalidValue can happen when specific joysticks are used. This issue
-	// will be fixed in GLFW 3.3.5. As a temporary fix, accept this error.
+	// will be fixed in GLFW 3.3.5. As a temporary fix, ignore this error.
 	// See go-gl/glfw#292, go-gl/glfw#324, and glfw/glfw#1763.
-	return acceptError(APIUnavailable, invalidValue)
+	err := acceptError(APIUnavailable, invalidValue)
+	if err.(*Error).Code == invalidValue {
```

Comment:
Is the comma-okay not needed? Won't the type conversion panic when `acceptError` returns a nil error?

Reply:
Already updated this PR to use comman-okay